mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule Crimpex.MixProject do
   def project do
     [
       app: :crimpex,
-      version: "0.1.1",
-      elixir: "~> 1.8",
+      version: "0.2.0",
+      elixir: "~> 1.16",
```

Comment:
You will need to change this to have it work on `master`

Reply:
What are you suggesting it should change to?

Reply:
Let's use `" ~> 1.14"`